### PR TITLE
feat: statsd deploy statistics

### PR DESCRIPTION
**Pull request by ireznice**

### CODE REVIEW

File: vc-worker.py [hunk 1]
```diff
@@ -13,10 +13,20 @@
...
 
+#def stats_increment_metric(metric):
+#    if Settings.statsd_client:
+#        try:
+#            Settings.statsd_client.incr(f"{Settings.app['statsd']['prefix']}.worker.actions.{metric}")
+#        except Exception as ex:
+#            Settings.raven.captureException(exc_info=True)
+#            logger.warning('stati failed: ', exc_info=True)
+
+
```
Comment (petrSchreiber) [inline vc-worker.py:29]: Fantastic improvement in the code, please just remove this commented out part.

**Review by petrSchreiber — CHANGES_REQUESTED** [review]
🙏 Please remove the commented section and I will merge the final version at lightspeed

**Review by petrSchreiber — APPROVED** [review]
🎖️ Significant improvement, thank you

**Review by timsta0 — APPROVED** [review]
tested on stage 👍

**Pull request merged**